packages/lib-taquito/index.ts

```diff
@@ -0,0 +1,111 @@
+import { DfnsApiClient, DfnsError } from '@dfns/sdk'
+import { Prefix, b58cencode, buf2hex, getPkhfromPk, hex2buf, mergebuf, prefix } from '@taquito/utils';
+import { Signer } from '@taquito/taquito'
+import { GetWalletResponse, GenerateSignatureResponse } from '@dfns/sdk/types/wallets'
+import { TxWatermark } from './constant'
+import { error } from 'console';
+
+export type DfnsWalletOptions = {
+  walletId: string
+  dfnsClient: DfnsApiClient
+  maxRetries?: number
+  retryInterval?: number
```

Comment:
we can remove these. not needed anymore.

Reply:
we can remove  `maxRetries` and `retryInterval`, no need to have those options anymore.